### ISSUE TITLE
Command-line interface?

### ISSUE THREAD

**Issue by dweindl** [ICB-DCM/pyPESTO#449]
Have there been any plans for a command-line interface for pypesto, e.g. `pypesto-optimize --problem=petab:problem.yaml --config=optimization_options.yaml --output=hdf5:results.h5`? I'd find that convenient... Any opinions?

Would require #117 to be implemented first.

**Comment by yannikschaelte** [comment]
No, but good idea. This would require all pypesto input options to be defined/definable in e.g. yaml files. Including the optimizer, optimizer options, global optimization options, sampler, problem bounds, ...

**Comment by FFroehlich** [comment]
> No, but good idea. This would require all pypesto input options to be defined/definable in e.g. yaml files. Including the optimizer, optimizer options, global optimization options, sampler, problem bounds, ...

would be nice anyways!
